public/sass/_variables.dark.scss

```diff
@@ -21,18 +21,17 @@ $green-shade: #23843b;
 // -------------------------
 $black: #000;
 $dark-1: #141414;
+$dark-1-5: #161719;
```

Comment:
Hm, not liking it much. Can we rescale the palette so that 1-5 becomes 2?

Reply:
Of course, this just wip. Probalbly gone name after lightness.